tree-viz.html

```diff
@@ -1,7 +1,7 @@
 <!doctype html>
 <head>
 <meta charset="UTF-8">
-<link rel="stylesheet" href="https://cdn.rawgit.com/martine/webtreemap/a2fa353833eca1a4821ba4c93fb6a7d546aed02a/webtreemap.css">
+<link rel="stylesheet" href="webtreemap.css">
```

Comment:
Is there an advantage to making this a plain file name rather than continuing the `INSERT webtreemap.css HERE` pattern? I think it's helpful to indicate that this is expected to be replaced.